### PR TITLE
Adding a new page for the business-case working group.

### PR DESCRIPTION
**Pull request by TheRealEdDawson**
Hello! Cameron Patience kindly asked me to create a page for the business case working group. So, here it is! :-)

### CODE REVIEW

File: content/en/working-group/business-case.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  - "[Kit Brown-Hoekstra](https://thegooddocs.slack.com/team/U02M65D4C7M)"
+  - "[macklin](https://thegooddocs.slack.com/team/U01DYRWG43X)"
+draft: false
+---
+
+In industry, we often find ourselves arguing for greater investment in documentation.
+
+We aim to discover the optimal information that could be used for a documentation business case, then create materials that help people make their own business case supporting documentation investment.
+
+We are currently identifying key statistics and metrics used by business decision makers, that could be used to support a proposal for increasing docs investment.
```
Comment (camerons) [inline content/en/working-group/business-case.md:28]: Nitpick (you can ignore): I'm a fan of [Timeless Documentation](https://developers.google.com/style/timeless-documentation), where aim to minimize maintenance by reducing points where we lock a doc to a point in time.
```suggestion
We are identifying key statistics and metrics used by business decision makers, that could be used to support a proposal for increasing docs investment. 
```

File: content/en/working-group/business-case.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+title: Business Case Working Group
+poc: "[Cameron Shorter](https://thegooddocs.slack.com/team/UKTGLQNGG)"
+meeting: Mondays at 12:00 p.m. Pacific Time, California (on Google Meet).
+status: Active
+description: Discover the optimal information for a documentation business case, and provide materials that help people make their own and succeed. For great justice.
```
Comment (camerons) [inline content/en/working-group/business-case.md:6]: Nitpick: (You can ignore). Some readers might not understand "For great justice."
```suggestion
description: Discover the optimal information for a documentation business case, and provide materials that help people make their own and succeed.
```

**Review by camerons — APPROVED** [review]
A couple of nitpick suggestions which you can incorporate or ignore as you see fit.

**Comment by TheRealEdDawson** [comment]
Thanks for the review, Cameron! I accepted your suggestions. :-)

**Pull request merged**